sklearn/neighbors/classification.py

```diff
@@ -161,7 +161,7 @@ def predict(self, X):
 
             for k, classes_k in enumerate(classes_):
                 mode = self._mode(_y[neigh_ind, k].toarray(), weights)
-                y_pred_sparse_multilabel.append(csr_matrix(mode).T)
+                y_pred_sparse_multilabel.append(csc_matrix(mode).T)
```

Comment:
Cool `y_pred_sparse_multilabel.append(csc_matrix(mode.T))`

Which test in `sklearn/cluster/tests/` should break when using `csc_matrix(mode).T`? I run `nosetests sklearn/cluster/` and all tests pass. Shall we add a regression test somwhere?